gelweb/gel2mdt/models.py

```diff
@@ -938,6 +965,10 @@ class MDT(models.Model):
         ('A', 'Active'), ('C', 'Completed')), default='A')
     gatb = models.NullBooleanField()
     sent_to_clinician = models.BooleanField(default=False)
+    gtab_made = models.BooleanField(default=False)
+    data_request_sent = models.BooleanField(default=False)
+    gtab_sent = models.BooleanField(default=False)
+    actions_sent = models.BooleanField(default=False)
 
```

Comment:
I don't think these have been added into the front end yet, maybe not needed yet

Reply:
Ignore me, its in there